awx/main/utils/licensing.py

```diff
@@ -379,10 +377,10 @@ def validate(self):
             return attrs
         attrs['valid_key'] = True
 
-        if Host:
```

Comment:
`if Host:`, but the `Host` here was the class `Host`. This should always be True, unless I'm crazy.